packages/env/src/constants/defaults.ts

```diff
@@ -29,7 +29,10 @@ export const DEFAULTS = {
 	RATE_LIMITER_GLOBAL_ENABLED: false,
 	RATE_LIMITER_GLOBAL_POINTS: 1000,
 	RATE_LIMITER_GLOBAL_DURATION: 1,
-	RATE_LIMITER_GLOBAL_STORE: 'memory',
+
+	RATE_LIMITER_REGISTRATION_ENABLED: false,
```

Comment:
Should this be enabled by default since we're basically handing the users a loaded gun here?

Reply:
Yeah thought the same thing, but we dont enable other rate limiters by default either. Both the normal and global one are disabled by default too. Mhh :thinking:

Reply:
This is to be compared to the login rate limiter which cannot be turned off either 👍

Reply:
Not every environment cares about limiting this so I'd keep the ability to turn it off but maybe we indeed should enable it by default with conservative defaults